recipe/meta.yaml

```
@@ -24,13 +24,15 @@ requirements:
 
   run:
     - python
-    - spyder >=5.0.5
+    - spyder >=5.1.1
     - tornado
     - terminado
     - coloredlogs
     - requests
 
 test:
+  commands:
```

```suggestion
  requires:
    - pip
  commands:
```

I think pip is not installed alongside Python in fresh envs on Linux, so this is required to make the new test pass.